generate_network.py

```diff
@@ -4,79 +4,68 @@
 import networkx as nx
 import random, math
 import json
-import argparse, os, sys
+import sys, os
+import string
+import typer
+from enum import Enum
+
+# Consts
+class nw_types(Enum):
```

Comment:
Classes and types in python are camel case. Unless internally the team agreed to use a different notation.